pkg/minikube/config/config.go

```diff
@@ -200,3 +201,29 @@ func (c *simpleConfigLoader) WriteConfigToFile(profileName string, cc *ClusterCo
 	}
 	return ioutil.WriteFile(path, contents, 0644)
 }
+
+// MultiNodeCNIConfig add default CNI config needed for multinode clusters and saves off the config
+func MultiNodeCNIConfig(cc *ClusterConfig) error {
+	if cc.KubernetesConfig.ExtraOptions.Get("pod-network-cidr", "kubeadm") == "" {
```

Comment:
what is this line doing ? is it getting two options at same time and check if they are "" ?

Reply:
ExtraOptions is multi-level: this returns the value of `pod-network-cidr` within the `kubeadm` component.